libs/MotionKit/include/MotionKit.hpp

```diff
@@ -47,9 +48,12 @@ class MotionKit
 	bool _stabilisation_requested  = false;
 	bool _rotate_x_turns_requested = false;
 
-	EulerAngles _euler_angles_previous {};
+	interface::EulerAngles _euler_angles_previous
+	{
+	};
 
-	const float kPIDMaxValue				  = 1.8F;
+	// ? Calculated with following PID parameters : ((270-180)*Kp + (360-180)*Kd)/KDeltaT
+	const float kEntryLimitSpeed			  = 3.15F;
```

Comment:
I don't understand the name of the variable

`entry` is usually used for rooms like `entry hall` (hall d'entrée)

I think you mean `input` (vs output)

```suggestion
const float kInputSpeedLimit = 3.15F;
```